CRM/Utils/PDF/Utils.php

```diff
@@ -315,7 +315,7 @@ private static function getCacheDir(array $settings): string {
       $cacheDir = $settings['font_dir'] . DIRECTORY_SEPARATOR . 'font_cache';
     }
     else {
-      $cacheDir = Civi::paths()->getPath('[civicrm.files]/upload/font_cache');
+      $cacheDir = CRM_Core_Config::singleton()->uploadDir . '/font_cache';
```

Comment:
For me `CRM_Core_Config::singleton()->uploadDir` already has a trailing slash but it doesn't seem to mind. There's probably lots of vars like that.